src/components/Escrow.js

```diff
@@ -7,9 +7,10 @@ import ethAddressConfig from "../abis/ethAddressConfig";
 import { tokenBalance1ABI } from "../abis/XY_Token";
 import TxnService from "../services/TxnService";
 import UserService from "../services/UserService";
+import { escrowABI } from "../abis/escrow_ABI";
 
 var bigInt = require("big-integer");
-const bigNumberMultiplier = 1000000000000000000;
+const bigNumberMultiplier = 1000000000;
```

Comment:
why is the value shrinked? is this multiplier value same for the all tokens -> MCT, MYT, MGT ?